Rubberduck.VBEEditor/QualifiedModuleName.cs

```diff
@@ -149,4 +149,44 @@ public override bool Equals(object obj)
             return !a.Equals(b);
         }
     }
+
+    public static class QualifiedModuleNameExtensions
```

Comment:
Why are the extensions in the same file as the struct itself? Either these should be part of the struct, which I actually would not prefer, or it should be in their own file in the `Extensions` namespace.

Reply:
I agree about the need for doc comments. I will add it. This is currently not in use. I wrote them up originally to help address the problems around the ownership when marshaling across the events. However, when I worked on the `EventArg`s classes, I found that we actually did not really need to marshal any SCW at all. I was able to achieve what I needed using the QMNs alone or in the case of the command bar, by copying some of its properties, since that was all what the callers needed. 

Should we run into a legitimate scenario where we need to marshal a SCW across the event, the intention is encourage the callers to take a dependency on the repository and take the SCW out there than directly via the `EventArg` implementations. 

The methods cannot be a part of the struct for reasons I can't remember. I left them in the same namespace primarily to avoid the need to add yet another namespace just to access the function and thus miss opportunities for where it may be used.